storages/storages.go

```diff
@@ -52,19 +52,40 @@ type Cluster struct {
 type Storages struct {
 	clustersConf     config.ClustersMap
 	backendsConf     config.BackendsMap
-	transport        http.RoundTripper
+	Transports       transport.Container
 	Clusters         map[string]NamedCluster
 	Backends         map[string]http.RoundTripper
 	lateRespHandler  transport.MultipleResponsesHandler
 	earlyRespHandler transport.MultipleResponsesHandler
 }
 
+// TransportRoundTripper for slecte
+type TransportRoundTripper struct {
```

Comment:
Why here? move it to transport package. Acctually don't move it, just implement RoundTrip method within transport Container. This package don't need to know anything about Container internals